locations/spiders/pijalniewedel.py

```diff
@@ -12,7 +13,11 @@
 
 class PijalnieWedel(Spider):
     name = "pijalniewedel_pl"
-    item_attributes = {"brand": "Pijalnia Czekolady E.Wedel", "brand_wikidata": "Q123251317"}
+    item_attributes = {
+        "brand": "Pijalnia Czekolady E.Wedel",
+        "brand_wikidata": "Q123251317",
+        "extras": Categories.CAFE.value,
```

Comment:
Does this also deserve some kind of cuisine=cake or similar? They look very dessert focused as a cafe.

Reply:
`cuisine=hot_chocolate` ? It is not the only thing, maybe they actually sell more coffee - but it is largest difference between them and other cafes.

And they put it directly into name.

Reply:
I set this to cafe because cuisine=hot_chocolate is so little used, and the cuisine can be inferred from the name. @iandees Can you advise please, thanks.

Reply:
I'm ok with just cafe for now.